backend/src/components/lookup.js

```diff
@@ -32,6 +32,28 @@ async function getRequestSubCategories() {
   return requestSubCategories
 }
 
+async function getFiscalYears() {
+  let fiscalYears = lookupCache.get('fiscalYears')
+  if (!fiscalYears) {
+    fiscalYears = []
+    let response = await getOperation('ofm_fiscal_years?$select=ofm_fiscal_yearid,ofm_caption,ofm_fiscal_year_number,ofm_start_date,ofm_end_date,statuscode,statecode&$orderby=ofm_fiscal_year_number')
+    response?.value?.forEach((item) => fiscalYears.push(new MappableObjectForFront(item, FiscalYearMappings)))
+    lookupCache.put('fiscalYears', fiscalYears, ONE_HOUR_MS)
+  }
+  return fiscalYears
+}
+
+async function getMonths() {
```

Comment:
I was going to ask my months were reference data but I see that we need monthId!

Reply:
yes, it is a lookup field in survey response entity in CRM